tensorflow/python/data/util/random_seed_test.py

```diff
@@ -64,19 +119,14 @@ def testRandomSeed(self):
       self.assertEqual((g_seed, op_seed), toutput, msg=msg)
       random_seed.set_random_seed(None)
 
+    check(tinput=tinput, toutput=toutput)
+
     if not context.executing_eagerly():
```

Comment:
Checks like these should be avoided when TF combinations are used. Instead, you should create separate test combinations for eager mode and graph mode.

Reply:
I have removed the redundant test and all the modes are covered in `default_test_combinations`